### PR TITLE
[E2E] Monitor tests are added

### PR DESCRIPTION
**Pull request by elenaNaboko**
The following tests are added:
-Verify that user can work with Monitor
-Verify that when user closes the Monitor by clicking on "Close Monitor" button Monitor stopped
-Verify that user can see the list of all commands from all clients ran for this Redis database in the list of results in Monitor
-Verify that when user reopens Monitor history is not displayed

### CODE REVIEW

File: tests/e2e/pageObjects/monitor-page.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        }
+    }
+    /**
+     * Start monitor function
+     */
+    async startMonitor(): Promise<void> {
+        await t.click(this.expandMonitor);
+        await t.click(this.startMonitorButton);
+        //Check for "info" command that is sent every 5 seconds
+        await t.wait(5000);
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/monitor-page.ts:65]: Can you explain?

File: tests/e2e/pageObjects/monitor-page.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        // await this.checkCommandInMonitorResults('info');
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/monitor-page.ts:66]: Redundant

File: tests/e2e/tests/critical-path/monitor/monitor.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    //Check the command from browser client
+    await monitorPage.checkCommandInMonitorResults(browser_command);
+    //Open Workbench page to create new client
+    await t.click(myRedisDatabasePage.workbenchButton);
+    //Send command in Workbench
+    await workbenchPage.sendCommandInWorkbench(workbench_command);
+    //Check that command from Workbench is displayed in monitor
+    await monitorPage.checkCommandInMonitorResults(workbench_command);
+    //Wait for info request
+    await t.wait(5000);
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/monitor/monitor.e2e.ts:80]: Why this, and not `{ timeout: 5000 }` within the function when you assert it?

File: tests/e2e/tests/critical-path/monitor/monitor.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import { addNewStandaloneDatabase } from '../../../helpers/database';
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/monitor/monitor.e2e.ts:1]: Are you sure you don't have import testcafe?

File: tests/e2e/pageObjects/monitor-page.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/monitor-page.ts]: BE automatically sends "info" command every 5 seconds. Starting monitor I am waiting 5 seconds just to get this command logged inside of monitor (just to make sure that monitor is started). 
I had "checkCommandInMonitorResults" function that checked if a specified command appeared in monitor (without timeouts)  but for the one "info" command used "t.wait". 
Fixed according to your comment (now { timeout: 5000 } is set for any command in the function "checkCommandInMonitorResults").

File: tests/e2e/pageObjects/monitor-page.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/monitor-page.ts]: Fixed

File: tests/e2e/tests/critical-path/monitor/monitor.e2e.ts (hunk 3 above)
Comment (elenaNaboko) [reply tests/e2e/tests/critical-path/monitor/monitor.e2e.ts]: Fixed

File: tests/e2e/tests/critical-path/monitor/monitor.e2e.ts (hunk 4 above)
Comment (elenaNaboko) [reply tests/e2e/tests/critical-path/monitor/monitor.e2e.ts:1]: I am not sure that understood your question. Could you please rephrase?

**Pull request merged**